src/Theme.jsx

```diff
@@ -61,35 +60,35 @@ export const Themes = {
  */
 function loadTheme(mode) {
   // https://mui.com/customization/color/#color-palette
-  const fontFamily = 'Roboto'
+  const fontFamily = 'Helvetica'
   const colors = {
     grey: {
       lightest: grey[100],
       light: grey[300],
-      medium: grey[500],
+      medium: grey[400],
       dark: grey[700],
       darkest: grey[900],
     },
     green: {
-      lightest: green[100],
+      lightest: '#CEE6CA',
```

Comment:
these are custom greys, the greys from the pallette are weird.

Reply:
Ok, I'll take some of these.  Palette is also about mutual contrast on many devices, so let's prefer them when possible.

Exceptions:
  - black should stay black because it's named
  - your switch from grey[700] to #444444 is barely different from grey[800] (#424242)
  - your grey medium is close to grey[400], i'll try that instead.